pkg/Directory.Build.props

```diff
@@ -33,6 +33,7 @@
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\lib\**\*" Pack="true" PackagePath="lib" />
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\analyzers\**\*" Pack="true" PackagePath="analyzers" />
     <Content Include="$(PackageAssetsPath)$(PackageIdFolderName)\runtimes\**\*" Pack="true" PackagePath="runtimes" />
+    <Content Include="$(PackageAssetsPath)..\..\..\THIRD-PARTY-NOTICES.TXT" Pack="true" PackagePath="\" />
```

Comment:
Let's use the `$(RepoRoot)` property instead:

`<Content Include="$(RepoRoot)THIRD-PARTY-NOTICES.TXT" Pack="true" PackagePath="\" />`

That way we don't have to do the directory assumption that `$PackageAssetsPath` is 3 down from the root, which might not always be true.

Reply:
Thanks